src/open_inwoner/utils/admin.py

```diff
@@ -62,6 +63,32 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class TimelineLogLevelFilter(admin.SimpleListFilter):
+    title = _("log level")
+    parameter_name = "log_level"
+
+    def lookups(self, request, model_admin):
+        return [(v, v) for v in logging.getLevelNamesMapping()] + [
+            ("unknown", _("Unknown"))
+        ]
+
+    def queryset(self, request, queryset):
+        if not (value := self.value()):
+            return queryset
+
+        if value == "unknown":
+            return queryset.filter(
+                ~Q(extra_data__has_key="log_level")
+                | Q(extra_data__log_level__isnull=True)
+            )
```

Comment:
Selecting `unknown` will return all logs which either (a) have the default log level `Info` (or whatever we choose in the future) because there's no custom log-level specified, or (b) have no log level at all. I wonder what the use case is for this kind of filter? Wouldn't it make more sense to _not_ filter and just show all the logs in this case?